tests/topotests/bgp_srv6l3vpn_sid/r1/vpnv6_rib_2001_8_srv6.json

```diff
@@ -0,0 +1,55 @@
+{
+  "1:30":{
+    "prefix":"2001:8::/64",
+     "advertisedTo":{
+      "2001::2":{
+        "hostname":"r2"
+      }
+    },
+    "pathCount":1,
+    "paths":[
+      {
+        "aspath":{
+          "string":"Local",
+          "segments":[],
+          "length":0
+        },
+        "nhVrfName":"vrf30",
+        "announceNexthopSelf":true,
+        "origin":"incomplete",
+        "metric":0,
+        "weight":32768,
+        "valid":true,
+        "sourced":true,
+        "local":true,
+        "bestpath":{
+          "overall":true,
+          "selectionReason":"First path received"
+        },
+        "extendedCommunity":{
+          "string":"RT:55:55"
+        },
+        "originatorId":"192.0.2.1",
+        "remoteLabel":48,
+        "remoteSid":"2002:db8:1:1::",
```

Comment:
I notice the `remoteSidStructure` field is missing here.

I assume the field is only missing from the topotest JSON file, but the "SRv6 SID Structure Sub-Sub-TLV" is still being announced by BGP.

Could you confirm this?